frontend/src/components/WorkProcessSection/index.jsx

```diff
@@ -14,43 +14,42 @@ const WorkProcess = () => {
     const { title, circleItem } = workProcessData;
 
     return (
-      <section
-        style={{ borderTopLeftRadius: '200px', marginTop: '5.625rem' }}
-        className="border-4 border-c1000 pt-24 pb-32 bg-c400  mt-20"
-      >
+      <section className="border-4 border-c1000 lg:pt-24 pt-16 pb-32 bg-c400 lg:rounded-tl-md lg:mt-24 mx-auto">
         <Heading
           as="h1"
           fontSize={HEADING_OPTIONS.FONT_SIZES.MEDIUM}
           color={HEADING_OPTIONS.COLOR.PRIMARY}
-          className="text-center mb-16"
+          className="text-center lg:mb-16 mb-16"
```

Comment:
what is the point here from using `lg:mb-16`, it's already token by the next class name `mb-16` you didn't change any value